pkg/odo/cli/pipelines/ui/ui.go

```diff
@@ -0,0 +1,246 @@
+		Help:    "By default images are built from source, whenever there is a push to the repository for your service source code and this image will be pushed to the image repository specified in this parameter, if the value is of the form <registry>/<username>/<repository>, then it assumed that it is an upstream image repository e.g. Quay, if its of the form <project>/<app> the internal registry present on the current cluster will be used as the image repository.",
+	}
+
+	err := survey.AskOne(prompt, &path, survey.Required)
+	ui.HandleError(err)
+
+	return path
+}
+
+func EnterInteractiveCommandLineOutputPath() string {
+	var path string
+	var prompt *survey.Input
+	prompt = &survey.Input{
+		Message: "Provide a path to write GitOps resources?",
+		Help:    "This is the path where the GitOps repository configuration is stored locally before you push it to the repository GitopsRepoURL <fill in the string from GitOpsRepoURL>",
+		Default: ".",
+	}
+
+	err := survey.AskOne(prompt, &path, nil)
+	ui.HandleError(err)
+
+	return path
+}
+
+func EnterInteractiveCommandLineGitWebhookSecret() string {
+	var path string
+	var prompt *survey.Input
+	prompt = &survey.Input{
+		Message: "Provide a secret that we can use to authenticate incoming hooks from your Git hosting service for the GitOps repository. (if not provided, it will be auto-generated)",
+		Help:    "The webhook secret is a secure string you plan to use to authenticate pull/push requests to the version control system of your choice, this secure string will be added to the webhook sealed secret created to enhance security. Choose a secure string of your choice for this field.",
+	}
+
+	err := survey.AskOne(prompt, &path, nil)
+	ui.HandleError(err)
+
+	return path
+}
+
+func EnterInteractiveCommandLineSealedSecrets() string {
+	var path string
+	var prompt *survey.Input
+	prompt = &survey.Input{
+		Message: "Name of the Sealed Secrets Services that encrypts secrets <sealed-secrets-controller>",
+		Help:    "If you have a custom installation of the Sealed Secrets operator, we need to know where to communicate with it to seal your secrets.",
+		Default: "sealed-secrets-controller",
+	}
+
+	err := survey.AskOne(prompt, &path, nil)
+	ui.HandleError(err)
+
+	return path
+}
+
+func EnterInteractiveCommandLineSealedSecretNamespace() string {
+	var path string
+	var prompt *survey.Input
+	prompt = &survey.Input{
+		Message: "Provide a namespace in which the Sealed Secrets operator is installed, automatically generated secrets are encrypted with this operator? <kube-system>",
+		Help:    "If you have a custom installation of the Sealed Secrets operator, we need to know how to communicate with it to seal your secrets",
+		Default: "kube-system",
+	}
+
+	err := survey.AskOne(prompt, &path, nil)
+	ui.HandleError(err)
+
+	return path
+}
+
+func EnterInteractiveCommandLineStatusTrackerAccessToken() string {
```

Comment:
Is it possible to skip InteractiveCommandLine from the func name and just use `EnterStatusTrackerAccessToken()`

Reply:
for this function or for all?